tests/benchmark/Dockerfile.in

```diff
@@ -1,6 +1,19 @@
 FROM ${FULL_BASE_IMAGE}
 
-RUN hg clone -r 9923b81a1d34 https://hg.python.org/benchmarks /app/benchmarks
-WORKDIR /app/benchmarks
-RUN python perf.py -r -b default /usr/bin/python2.7 /usr/bin/python3.4
-RUN python perf.py -r -b default /usr/bin/python3.4 /opt/python3.5/bin/python3.5
+# Install performance
+RUN python2 -m pip install performance
+
+# Create virtual environment
+RUN pip install --upgrade virtualenv
```

Comment:
So, I'd recommend being consistent here - either both this an line 4 should use `pip install` or both should use `python2 -m pip`.